src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/ApplyVQSR.java

```diff
@@ -0,0 +1,628 @@
+ *
+ * <h3>Caveats</h3>
+ *
+ * <ul>
+ * <li>The tranche values used in the example above are only meant to be a general example. You should determine the level of sensitivity
+ * that is appropriate for your specific project. Remember that higher sensitivity (more power to detect variants, yay!) comes
+ * at the cost of specificity (more false negatives, boo!). You have to choose at what point you want to set the tradeoff.</li>
+ * <li>In order to create the tranche reporting plots (which are only generated for SNPs, not indels!) Rscript needs to be
+ * in your environment PATH (this is the scripting version of R, not the interactive version).</li>
+ * </ul>
+ */
+@CommandLineProgramProperties(
+        summary = "Apply a score cutoff to filter variants based on a recalibration table",
+        oneLineSummary = " Apply a score cutoff to filter variants based on a recalibration table",
+        programGroup = VariantProgramGroup.class
+)
+public class ApplyVQSR extends VariantWalker {
+
+    protected static final String LOW_VQSLOD_FILTER_NAME = "LOW_VQSLOD";
+    private final double DEFAULT_VQSLOD_CUTOFF = 0.0;
+
+    private boolean foundSNPTranches = false;
+    private boolean foundINDELTranches = false;
+
+    /////////////////////////////
+    // Inputs
+    /////////////////////////////
+    @Argument(fullName="recal_file", shortName="recalFile", doc="The input recal file used by ApplyRecalibration", optional=false)
+    private FeatureInput<VariantContext> recal;
+
+    @Argument(fullName="tranches_file", shortName="tranchesFile", doc="The input tranches file describing where to cut the data", optional=true)
+    private File TRANCHES_FILE;
+
+    /////////////////////////////
+    // Outputs
+    /////////////////////////////
+
+    @Argument(fullName= StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName=StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="The output recal file used by ApplyRecalibration", optional=false)
+    private String output;
+
+    /////////////////////////////
+    // Command Line Arguments
+    /////////////////////////////
+    @Argument(fullName="ts_filter_level", shortName="ts_filter_level", doc="The truth sensitivity level at which to start filtering", optional=true)
+    private Double TS_FILTER_LEVEL = null;
+
+    /**
+     *  Filter the input file based on allele-specific recalibration data.  See tool docs for site-level and allele-level filtering details.
+     *  Requires a .recal file produced using an allele-specific run of VariantRecalibrator
+     */
+    @Argument(fullName="useAlleleSpecificAnnotations", shortName="AS", doc="If specified, the tool will attempt to apply a filter to each allele based on the input tranches and allele-specific .recal file.", optional=true)
+    private boolean useASannotations = false;
+
+    @AdvancedOption
```

Comment:
They look redundant. I removed all references to the "option"-suffixed ones from this PR and created a ticket to remove the existing usages and remove the containing package. https://github.com/broadinstitute/gatk/issues/2130.